tests/.infra/crw-ci/single-host/k8s/Jenkinsfile

```diff
@@ -0,0 +1,176 @@
+pipeline {
```

Comment:
This is single-host version of tests/.infra/crw-ci/nightly/k8s/Jenkinsfile

Let's follow pipelines directory structure logic:
- tests/.infra/crw-ci/nightly/k8s/single-host/Jenkinsfile
- tests/.infra/crw-ci/nightly/k8s/multi-host/Jenkinsfile

Reply:
Fixed.